package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Denormalizer for normalizr",
   "main": "lib/index.js",
   "files": [
+    "dist",
     "lib"
```

Comment:
> Why one version 

An UMD build works for CommonJS, AMD and global var, so I wouldn't mind to include this dist even for the standard lib. Anyway if this is the common practice then i can live with it 😄